.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode/settings.json
 log.txt
+touch
```

Comment:
actually this a bug that script make an extra file name ```touch```
we have to find the root cause.